go.mod

```diff
@@ -5,10 +5,9 @@ go 1.12
 require (
 	github.com/golang/protobuf v1.3.2
 	github.com/grpc-ecosystem/grpc-gateway v1.9.4
-	github.com/gunk/gunk v0.1.1-0.20190611084741-4777ff336fd8
-	github.com/gunk/opt v0.0.0-20190514110406-385321f21939
```

Comment:
is opt no longer necessary? we still import it: https://github.com/openbank/openbank/blob/9fbb6c1bf7f5754c598cd195f7d6f613bdb219d8/v1/accounts/accounts.gunk#L54

Reply:
I ran go mod tidy and they are removed, probably nothing directly import it. Will add it to tools.go to prevent this.

Reply:
added opt/http to tools.go and re-ran go mod tidy.